### PR TITLE
Readme DUG #6 update

### PR DESCRIPTION
**Pull request by wiktormowinski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ Following mechanisms may be used for DUT control:
 
 ![regression-architecture](https://cloud.3mdeb.com/index.php/s/KkERgGoniBtjfC4/preview)
 
+### Current osfv_cli purpose
```
Comment (macpijan) [inline README.md:46]: I'd call this OSFV architecture

File: README.md [hunk 2]
```diff
@@ -43,6 +43,10 @@ Following mechanisms may be used for DUT control:
...
+
+![](./docs/img/osfv_cli_after.png)
```
Comment (macpijan) [inline README.md:48]: Można dać ze dwa zdania kontekstu pod tym.

Że nie używamy obecnie repozytoriów takich jak:
- https://github.com/3mdeb/sonoff-rest-api
- https://github.com/3mdeb/rtectrl-rest-api

Zamiast tego, używamy wspólnych bibliotek z: https://github.com/Dasharo/osfv-scripts/tree/main/osfv_cli/src/osfv/libs

Są one używane jednocześnie w `osfv_cli` jak i testach OSFV.

Biblioteki RF korzystające ze wspólnych keywordów są tutaj: https://github.com/Dasharo/osfv-scripts/tree/main/osfv_cli/src/osfv/rf

A wszystko to do środowiska dołączane jest przez `requirements.txt` - należy więc dbać o jego aktualność.

File: README.md (hunk 1 above)
Comment (wiktormowinski) [reply README.md]: title updated: https://github.com/Dasharo/open-source-firmware-validation/pull/459/commits/ede9434d7fa2744836168f82ed751eae806ecb54

File: README.md (hunk 2 above)
Comment (wiktormowinski) [reply README.md:48]: info included: https://github.com/Dasharo/open-source-firmware-validation/pull/459/commits/ede9434d7fa2744836168f82ed751eae806ecb54

Comment (macpijan) [reply README.md:48]: @wiktormowinski please fix according to offline discussion

Comment (wiktormowinski) [reply README.md:48]: sure thing: https://github.com/Dasharo/open-source-firmware-validation/pull/459/commits/b80963d585c0baafde69e70b70fe9c8d5682dcc8

**Comment by JanPrusinowski** [comment]
In Macieks presentation there asre mensioned things like DCU and generating documentation from code. Shouldn't it also be included within this PR?

**Comment by JanPrusinowski** [comment]
So to be clear. Everything regarding part of `Recent improvements` should be added in this PR (if not present in README already).
- Platform-configs rework <- reduced code duplication across platforms etc. The present documentation in this area should be updated to the new standard,
- Automatic documentation generation
- Switch Sonoff API
- DCU Integration <- Its not ready yet, but an information of what it is going to be and how it will work is explained by Maciek and can be added

To consider: - At the end Piotr states what is the goal of OSFV. Would be nice to see what the target is in OSFV developmnet and what we aim to achieve.

**Comment by wiktormowinski** [comment]
ok @JanPrusinowski ,
- https://github.com/Dasharo/open-source-firmware-validation/pull/459/commits/1dbfbd1860b679cb573c27db598a34a30dcf8fa7
- i don't really know what information about the sonoff update really include its information more meant for changelog if anything 
- already there
- https://github.com/Dasharo/open-source-firmware-validation/pull/459/commits/269ae00c9b679bc200a0de516ffb2dc939f19283

quick thought: I dont like the idea of adding everything to that one singular readme, if we want to describe something in detail i think it should be included in smaller files in docs directory or even dasharo docs repo and just linked here so that you can access more info

**Review by JanPrusinowski — APPROVED** [review]

**Pull request merged**